src/d3d12/d3d12_resource_pool_texture.hpp

```diff
@@ -2,9 +2,15 @@
 
 #include "../resource_pool_texture.hpp"
 #include "d3d12_structs.hpp"
+#include <DirectXMath.h>
 
 namespace wr
 {
+	struct MipMapping_CB
```

Comment:
Useless struct. Could just write `DirectX::XMFLOAT2`. Unless you expect to expand this.

Reply:
I'm expecting to expand this, when implementing support for non-power of 2, rectangular and srgb textures. Right now the CB is not even used, it will be then.